composer.json

```diff
@@ -12,6 +12,12 @@
             "test\\eLife\\Journal\\": "test/"
         }
     },
+    "repositories": [
+        {
+            "type": "path",
+            "url": "../patterns-php"
+        }
+    ],
```

Comment:
This is unnecessary

Reply:
I see, you're using it this was so it can use your local copy of patterns-php. Obviously, we need to remove this for CI.